.github/ISSUE_TEMPLATE/enhancement-proposal.md

```diff
@@ -0,0 +1,22 @@
+---
+name: OpenEBS Enhancement Proposal [ OEP ]
+about: Create the tracking parent issue for an OEP
+
+---
+
+### Enhancement Description
+
+- Clear and concise enhancement description (one-liner):
+- OpenEBS Enhancement ProposalOpenEBS Enhancement Proposal: <!-- link to oep file; if none yet, link to PR -->
+- Discussion Link: <!-- link to any issue, thread, or recording where the Enhancement was discussed before OEP creation -->
```

Comment:
The comments are helpful but don't render to the user. Should we instead make them visible?